definitions/divorce/json/CaseRoles-shareACase.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "18/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "[caseworker-caa]",
```

Comment:
I think this case role will come into play later. Or actually we might not need to implement it as I think the functionalities of the caseworker-caa are already defined in ExUI. I could be wrong on this but I think it's right 😅

Reply:
Yeah i spoke to harry about this yesterday. I had reached the part where i believed that the role was needed but wasnt sure on how to impliment it just yet

Reply:
Actually it should have "CRU" authorisation in "AuthorisationField..."
` {
    "LiveFrom": "16/03/2020",
    "CaseTypeID": "DIVORCE",
    "CaseFieldID": "PetitionerOrganisationPolicy",
    "UserRole": "caseworker-caa",
    "CRUD": "CRU"
  }`

Reply:
Thank you!